Gems/LmbrCentral/Code/Source/Shape/EditorTubeShapeComponentMode.cpp

```diff
@@ -52,6 +58,85 @@ namespace LmbrCentral
         AZ::TransformNotificationBus::Handler::BusDisconnect();
     }
 
+    void EditorTubeShapeComponentMode::Reflect(AZ::ReflectContext* context)
+    {
+        if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serializeContext->Class<EditorTubeShapeComponentMode, EditorBaseComponentMode>(AZ::Internal::NullFactory::GetInstance());
+        }
+    }
+
+    void EditorTubeShapeComponentMode::RegisterActions()
+    {
+        auto actionManagerInterface = AZ::Interface<AzToolsFramework::ActionManagerInterface>::Get();
+        AZ_Assert(actionManagerInterface, "EditorTubeShapeComponentMode - could not get ActionManagerInterface on RegisterActions.");
+
+        auto hotKeyManagerInterface = AZ::Interface<AzToolsFramework::HotKeyManagerInterface>::Get();
+        AZ_Assert(hotKeyManagerInterface, "EditorTubeShapeComponentMode - could not get HotKeyManagerInterface on RegisterActions.");
+
+        // Duplicate Vertices
+        {
+            constexpr AZStd::string_view actionIdentifier = "o3de.action.tubeShape.resetRadii";
+            AzToolsFramework::ActionProperties actionProperties;
+            actionProperties.m_name = s_resetRadiiTitle;
+            actionProperties.m_description = s_resetRadiiDesc;
+            actionProperties.m_category = "Tube Shape";
+
+            actionManagerInterface->RegisterAction(
+                EditorMainWindowActionContextIdentifier,
+                actionIdentifier,
+                actionProperties,
+                []
+                {
+                    /*
```

Comment:
Nit: I presume this dead code will be removed but just in case... ;p

Reply:
Yup I need to replace it with a bus call, incoming.

Reply:
Fixed